- Indicator new listing items.

cis-shubham-t opened the pull request:
Task: https://particl.atlassian.net/projects/PBD/issues/PBD-94


Base PR: https://github.com/particl/particl-desktop/pull/1342

Continued on the @EagleEye25 's PR and create a v2 branch here

Commit:
added functionality to check for new listings

Commit:
minor improvements

cis-shubham-t committed:
- remove the newListingsCheck() and use the loadPage().

cis-shubham-t committed:
- update PR with market branch.

cis-shubham-t committed:
- Added the class for the indicator.
- Added the mechanism for checking the new listing comming or not.

cis-shubham-t commented:
@AllienWorks  added the 'indicator' class.
if the new listing arrives then that case is added automatically.

![indicator](https://user-images.githubusercontent.com/38350953/56294692-1edfc880-6149-11e9-80bf-d28a13717cf1.png)

Please add the CSS to show the indicator effect or indicator button.

Thanks

AllienWorks committed:
Listings: indicator for new listings styled

AllienWorks commented:
Styled.

@cis-shubham-t can we switch the tooltip to "New listings available" when, ehm, new listings are available (instead of "Refresh listings")? :)

cis-shubham-t committed:
- Change the tooltip if new listing is arrieves the tooltip will be 'New listings available' instead of 'Refresh listings'

cis-shubham-t committed:
- Update the check buy which blicker is blick for every case.

cis-shubham-t committed:
- minor updates

zaSmilingIdiot committed:
Merge branch 'market' of https://github.com/particl/particl-desktop into indicate-new-listingItems-v2

zaSmilingIdiot committed:
Listings Overview updates to show indicator for new (undisplayed) listings:
* moves function to check for new possible listings into the loadPage function.
* increases timeout duration to 20 seconds (although I believe increasing it to 60 seconds should be sufficient as well)
* removes use of array to store entire first page listings: replaces it with a string variable as its only the first listing hash we're actually interested in, so better to store just that
* adds checks to ensure in flight requests are not processed
* renames some vars
* ensures that a change of filters properly resets the hash check (so as not to indicate new items just because the current filter changed)

zaSmilingIdiot committed:
Fixes linting issue (missing typedef).

zaSmilingIdiot committed:
Update listings.component.ts

Ensures that the indicator for new items is not reset when reaching page 0 from a scroll up/back; the rest only occurs when requesting the results are reloaded.
